frontend/src/pages/HomePage.js

```diff
@@ -54,12 +51,9 @@ function HomePage() {
   );
 
   useEffect(() => {
-    if (loading) {
-      fetchPosts();
-      fetchUsers();
-      setLoading(false);
-    }
-  }, [fetchPosts, fetchUsers, loading]);
+    fetchPosts();
+    fetchUsers();
+  }, []);
```

Comment:
@Mr-spShuvo I need help understanding how to fix an eslint rule.  The linter is saying that we should put `[fetchPosts, fetchUsers]` inside the deps array, but when I do, the component infinitely loops because the fetchPosts method is re-created each render.  It feels dirty not fixing the lint warning, but at the same time, this code is so much cleaner with just having the empty array of deps `[]`.

Reply:
Yeah, eslint pretty  clingy about hook dependencies. If it throws you a warning you are not using the hook properly. I will simplified the code and a PR. Look there.

Reply:
#183